include/fault.h

```diff
@@ -51,15 +54,28 @@ NORETURN void Fault_AddHungupAndCrash(const char* file, int line);
 void Fault_AddClient(FaultClient* client, void* callback, void* arg0, void* arg1);
 void Fault_RemoveClient(FaultClient* client);
 
+#if FAULT_VERSION == FAULT_OOTGC
 void Fault_AddAddrConvClient(FaultAddrConvClient* client, void* callback, void* arg);
 void Fault_RemoveAddrConvClient(FaultAddrConvClient* client);
+#endif
 
 // For use in Fault Client callbacks
 
-void Fault_WaitForInput(void);
-void Fault_FillScreenBlack(void);
 void Fault_SetFrameBuffer(void* fb, u16 w, u16 h);
 
+#if FAULT_VERSION == FAULT_OOTN64
+
+void Fault_SetCursor(s32 x, s32 y);
+void Fault_Printf(const char* fmt, ...); // TODO return type?
```

Comment:
this does match as returning s32 (using `s32 ret = _Printf(...); ... return ret;`)